src/components/DateField/DateField.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { useDeskproAppTheme } from "@deskpro/app-sdk";
+import { H1, Input, Stack, Label as UILabel } from "@deskpro/deskpro-ui";
+import { faCalendarDays } from "@fortawesome/free-solid-svg-icons";
+import { FC } from "react";
+import { DateTimePickerProps } from "react-flatpickr";
+import styled from "styled-components";
+import { DateTimePicker } from "../DateTimePicker/DateTimePicker";
+import "./DateField.css";
+
+const LabelDate = styled(UILabel)`
+  margin-top: 5px;
+`;
+
+export type MappedFieldProps = DateTimePickerProps & {
+  id?: string;
+  label: string;
+  error?: boolean;
+  value: string;
+  onChange: (date: [Date]) => void;
+};
+
+const DateInput = styled(Input)`
+  :read-only {
+    cursor: pointer;
+  }
+`;
+
+export const DateField: FC<MappedFieldProps> = ({
```

Comment:
Please use `DateInput` component from `app-sdk`

Reply:
date input does not have a label property, I need it

Reply:
There is also no label in the input components from `desktop-ui`, because it's just an input component.

You can do like this:
```jsx
<Label>
    <DateInput />
</Label>
```

Reply:
check my component, it does alot more than that.

Reply:
so let's improve the common component in the `app-sdk`